dist/app/modules/MainScreen/styles.js

```diff
@@ -0,0 +1,14 @@
+// import styled from 'styled-components';
+import { styled } from '../../styles/themes/defaultTheme';
+export const MainWrapper = styled.View `
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+`;
+export const MainTitle = styled.Text `
```

Comment:
Please create some generic Text component which will have some general styling like font-family etc. and specific components inside modules should inherit from it.

Reply:
something like 

export const MainTitle = styled(MyGenericText)